app/models/health/qualifying_activity.rb

```diff
@@ -60,7 +85,19 @@ class QualifyingActivity < HealthBase
     end
 
     scope :face_to_face, -> do
-      where(mode_of_contact: :in_person)
+      @face_to_face_query ||= begin
```

Comment:
scopes are class methods, are you sure we want to cache this?  It will only be calculated at boot.

Reply:
That looks ok, but also maybe unnecessary unless we're calling the scope in a tight loop.

Reply:
Yeah -- probably unnecessary, but I figured, why do it more than once... Probably it could be meta-programmed, but that seems more confusing.